modules/framework/src-jvm/RunnerCompat.scala

```diff
@@ -119,9 +120,15 @@ trait RunnerCompat[F[_]] { self: sbt.testing.Runner =>
                   gate)
     stillRunning.set(sbtTasks.size)
 
+    val maybeTimeout: Option[FiniteDuration] = Try(System.getProperty("weaver.test.startupTimeout"))
```

Comment:
pro-tip : you can use `sys.props` (which gives you a scala map of the available system properties)